graphql-schemas/schema.yml

```diff
@@ -951,6 +951,41 @@ confs:
       schema: /aws/policy-1.yml
       subAttr: account
 
+- name: CNAProvisioner_v1
+  interface: ExternalResourcesProvisioner_v1
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: name, type: string, isRequired: true, isUnique: true, isSearchable: true }
+  - { name: description, type: string }
+  - { name: ocm, type: OpenShiftClusterManager_v1, isRequired: true }
+
+- name: NamespaceCNAsset_v1
+  interface: NamespaceExternalResource_v1
+  fields:
+  - { name: provider, type: string, isRequired: true }
+  - { name: provisioner, type: CNAProvisioner_v1, isRequired: true }
+  - { name: resources, type: CNAsset_v1, isRequired: true, isList: true, isInterface: true }
+
+- name: CNAsset_v1
```

Comment:
Why it's called `CNAsset` and not `CNAAsset`? The same applies for `NamespaceCNAsset_v1`

Reply:
CNA stands for Cloud Native Assets. CNAAssets felt weird, so I went with `CNAsset` :D

Reply:
Ok :)